lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -461,7 +465,6 @@
       mode:
         description:
           - What publish mode to use.
-          - Service will be removed and recreated when changed.
```

Comment:
Is this no longer true for `publish` mode?

Reply:
This was never true. In https://github.com/ansible/ansible/pull/52634/ I added that string to the wrong `mode`-option.

Reply:
It's supposed to be documented on the actual `mode`-option and not `publish.mode`.